script.js

```diff
@@ -16,3 +16,129 @@ menuLinks.forEach((item) => {
+    type: ['FACEBOOK', 'Full Stack Dev', '2015'],
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    technologies: ['HTML', 'CSS', 'Javascript'],
+  },
+  {
+    id: 'Three',
+    img: {
+      src: './desktopPics/newsnap.png',
+      alt: 'Third Project demo',
+    },
+    title: 'Facebook 360',
+    type: ['CANOPY', 'BACKEND DEV', '2015'],
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    technologies: ['HTML', 'CSS', 'Javascript'],
+  },
+  {
+    id: 'Four',
+    img: {
+      src: './Worksection Pics/dsk4.png',
+      alt: 'Fourth Project demo',
+    },
+    title: 'Uber Navigation',
+    type: ['UBER', 'Lead Developer', '2018'],
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    technologies: ['HTML', 'CSS', 'Javascript'],
+  },
+];
+
+const popMain = document.querySelector('.pop-section');
+
+function popUp() {
+  workfiles.forEach((files) => {
+    popMain.innerHTML += `
+
+    <div class="pop-container" 
+    files-target="${files.id}">
+    <i class="fa fa-close" style="font-size:24px"></i>
+    <h2>${files.title}</h2> 
+
+
+    <ul class="pop-tech">
+    <li class="canopy">${files.type[0]}</li>
+    <li>${files.type[1]}</li>
+    <li>${files.type[2]}</li>
+  </ul>
+  <div class="preview-img">
+    <img class="pop-pic" src="${files.img.src}" alt="${files.img.alt}" />
+    <div class="pop-info">
+      <p>
+        ${files.description}
+       
+      </p>
+      <div class="pop-view-2">
+        <ul class="pop2-lang">
+          <li>${files.technologies[0]}</li>
+          <li>${files.technologies[1]}</li>
+          <li>${files.technologies[2]}</li>
+        </ul>
+        <hr>
+        <div class="pop-button">
+          <button><a href="https://studiosnabeel.github.io/">See Live</a><img src="./Pictures/Social/icon-see-live.png"></button>
+          <button><a href="https://github.com/studiosnabeel/studiosnabeel.github.io">See Source</a><img src="./Pictures/Social/icon-see-source.png"></button>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
- [ ] Good job working on your portfolio! On the desktop view, kindly ensure that you use the correct margin/ padding-top which will ensure that the vertical scroll that is currently there will be removed.
- [ ] Also, I have realized that the color of the paddings from the edges of the screen to your popup do not match the one that is in the Figma design. Yours has a darker shade than the one that is in the Figma design and distorts the design view. Kindly ensure that you use the correct design colors.

**This is the dark section I mean:**

![image](https://user-images.githubusercontent.com/86318284/196884882-69bf6590-7a8a-4f8a-9ebf-3bbf2ebf3e08.png)

**This is how I am seeing your popup on desktop view:**

[Screencast from 20-10-2022 10:32:48 ASUBUHI.webm](https://user-images.githubusercontent.com/86318284/196885088-1f0a3417-2760-4607-961b-0b9f548c3060.webm)

**This is the desired view:**

![image](https://user-images.githubusercontent.com/86318284/196885170-d41b9483-e51b-4500-8b87-1c7fe1963ef2.png)
 
**Note:** To better match the design guidelines, you could blur the background so that it matches the Figma design. Have a look at [this resource](https://www.w3schools.com/cssref/css3_pr_backdrop-filter.asp) to learn how to achieve this.

Reply:
Blur was added to the background as suggested. Scroll removed successfully and designed desktop version according to Figma design ✔👍❤